src/Quartz/Core/QuartzScheduler.cs

```diff
@@ -264,7 +264,7 @@ protected QuartzScheduler()
         /// properties.
         /// </summary>
         /// <seealso cref="QuartzSchedulerResources" />
-        public QuartzScheduler(QuartzSchedulerResources resources, TimeSpan idleWaitTime) : this()
+        internal QuartzScheduler(QuartzSchedulerResources resources, TimeSpan idleWaitTime) : this()
```

Comment:
Can I ask why the visibility of this constructor was changed? This breaks my code and makes this class impossible to derive from given that `resources` is assumed to be not-null but I can't see a way to set it without the visibility of this constructor being restored.

Reply:
Can you show the code?

Reply:
I just create an instance of the scheduler using a custom configured `QuartzSchedulerResources`, `new QuartzScheduler(resources, waitTime);`. Aside from how I am using this class, with this change, this class becomes impossible to derive from. It has an empty protected constructor, but developers can't actually use that since that would leave `resources` as null, which the rest of the class assumes that it isn't.